source/components/ui/msm-modal/modal-ok-cancel.html

```diff
@@ -0,0 +1,27 @@
+<div class="modal-header modal-desktop">
```

Comment:
I wonder why the modal stuff is already in master as there is still an open PR. I'll not repeat the comments for that in here.